<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MW_SIN" name="f1" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollBar" name="hsb_f1">
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox { border: 0;  }</string>
      </property>
      <property name="title">
       <string>功能區</string>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>20</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>其他功能</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QCheckBox" name="followF1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>追蹤最新</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_show">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>顯示輔助線</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>顯示資料寬度</string>
       </property>
       <widget class="QPushButton" name="btn_f1plus">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QLabel" name="lal_f1Width">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Width: 3</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_f1sub">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Timer Control</string>
       </property>
       <widget class="QPushButton" name="btn_timerStart">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>START!</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_timerStop">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>STOP!</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MW_SIN</class>
   <extends>QWidget</extends>
   <header>MatplotlibWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
